crowbar_framework/app/models/api/storage.rb

```diff
@@ -15,6 +15,32 @@
 #
 
 module Api
-  class Storages
+  class Storage < Tableless
+    def repocheck
+      {}.tap do |ret|
+        ret["ceph"] = {}
+        ceph_node_architectures = NodeObject.find(
+          "roles:ceph-mon AND ceph_config_environment:*"
```

Comment:
Ah, you re looking for arch, not system version. Hm. Actually I have no idea about clouds with possible different architectures...